CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace CompanyEmployees.Core.Domain.Entities
```

Comment:
The `Salary` property is currently non-nullable (`decimal`) but may not always be initialized. Consider making it nullable (`decimal?`) or ensuring it's always set to avoid runtime issues.

```suggestion
public decimal? Salary { get; set; }
```